src/main/java/it/eng/idsa/businesslogic/service/impl/DapsTokenProviderServiceImpl.java

```diff
@@ -47,7 +47,7 @@ public class DapsTokenProviderServiceImpl implements DapsTokenProviderService {
 	public String provideToken() {
 		logger.info("Requesting token");
 		if(!useDaps) {
-			logger.info("Daps not configured but must use some token - using value {}", UtilMessageService.TOKEN_VALUE);
+			logger.info("Daps not configured - continuing without token");
```

Comment:
We should say that we will use dummy token value, since token is mandatory and we cannot go without it.
Just say - continuing with dummy token value (no need to print actual dummy value)